cypress/integration/specialFeatures/scriptSwitching/testsForAMPOnly.js

```diff
@@ -0,0 +1,3 @@
+export default ({ serviceName, pageType, path }) => {
```

Comment:
Shall we add a comment here, just to state that this type of user journey is not applicable for AMP, with some brief details from the slack chat?

Reply:
👍